.github/workflows/website.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Install R packages
         run: |
           Rscript -e "install.packages(c('factoextra'))"
+	  Rscript -e "install.packages(c('quarto'))"
```

Comment:
this should not be mandatory. quarto is available by default as an external program

Reply:
ok I see you didn't respect the space formatting